### PR TITLE
fix: change jwt parse function

### PR DESCRIPTION
**Pull request by yonatandt**
Signed-off-by: Yonatan <yonatald@gmail.com>

### CODE REVIEW

File: src/app/service/user.service.ts [hunk 1]
```diff
@@ -16,12 +16,19 @@ export class UserService {
...
 
 		return null;
-	}
+  }
 }
+
+function parseJwt (token) {
+  var base64Url = token.split('.')[1];
+  var base64 = base64Url.replace(/-/g, '+').replace(/_/g, '/');
+  var jsonPayload = decodeURIComponent(atob(base64).split('').map(function(c) {
```
Comment (omrishtam) [inline src/app/service/user.service.ts:29]: There's `atob(base64)` so this still wouldn't work...

**Review by omrishtam — CHANGES_REQUESTED** [review]

File: src/app/service/user.service.ts (hunk 1 above)
Comment (yonatandt) [reply src/app/service/user.service.ts:29]: I checked this function inside and it worked

Comment (omrishtam) [reply src/app/service/user.service.ts:29]: K

**Review by omrishtam — APPROVED** [review]

File: src/app/service/user.service.ts (hunk 1 above)
Comment (yonatandt) [reply src/app/service/user.service.ts:29]: The problem is probably because of `window.atob`

**Pull request merged**